test/api/models/utils/JsonErrorValidators.scala

```diff
@@ -45,9 +45,11 @@ trait JsonErrorValidators {
 
   implicit class JsResultOps[T](res: JsResult[T]) {
 
-    def errors: JsErrors = res match {
-      case JsError(jsErrors) => jsErrors
-      case JsSuccess(_, _)   => fail("A JSON error was expected")
+    def errors: JsErrors = {
```

Comment:
can remove the extra curly brackets the have been added here and it's more concise + explicit to not have them